### PR TITLE
clarify the implicit parameters of AskPattern, #28836

### PR DESCRIPTION
**Pull request by patriknw**
References #28836

### CODE REVIEW

File: akka-actor-typed-tests/src/test/scala/docs/akka/typed/InteractionPatternsSpec.scala [hunk 1]
```diff
@@ -426,6 +428,8 @@ class InteractionPatternsSpec extends ScalaTestWithActorTestKit with AnyWordSpec
     // asking someone requires a timeout if the timeout hits without response
     // the ask is failed with a TimeoutException
     implicit val timeout: Timeout = 3.seconds
+    // implicit ActorSystem in scope
+    implicit val system: ActorSystem[_] = theSystem
```
Comment (renatocaval) [inline akka-actor-typed-tests/src/test/scala/docs/akka/typed/InteractionPatternsSpec.scala:432]: how was this compiling without one in scope?

**Review by octonato — APPROVED** [review]

File: akka-actor-typed-tests/src/test/scala/docs/akka/typed/InteractionPatternsSpec.scala (hunk 1 above)
Comment (patriknw) [reply akka-actor-typed-tests/src/test/scala/docs/akka/typed/InteractionPatternsSpec.scala:432]: it's in scope by the testkit

**Review by johanandren — APPROVED** [review]

**Comment by patriknw** [comment]
looks like we have a mima problem in master, I will fix

**Comment by johanandren** [comment]
PLS BUILD

**Review by chbatey — APPROVED** [review]

**Pull request merged**